src/molecules/Modal/modal.module.scss

```diff
@@ -24,6 +24,14 @@ $border: 1px solid color('neutral-5');
       padding: 0;
       border-bottom: none;
     }
+
+    .section:not( :first-child ) {
```

Comment:
I know we're trying to avoid using `not`, but I don't think there's a better way to do this, unless we use `> *` which we are also trying to avoid